catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,546 @@
+local id = 'USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/TRENDS_YEAR';
+local subdir = 'USGS';
+local version = '5';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO(AG): Remove skip_indexing when the preview image is ready.
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'RCMAP Rangeland Component Timeseries V5 Trends Year (1985-2021)' + version,
+  version: version,
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    Currently available yearly trends statistics for 1985-2021.
+

```

Comment:
@simonff are you able to see the error here? I've tried a lot of different things and can't seem to fix it. Thanks!

Reply:
I've tried fixing it 8 times today. Let me know how I can fix this in the description. Thank you!

Reply:
Thank you. I redid the description. I pulled it into both Notepad++ and Word to check the spacing and characters. I refitted the width to 80 characters or less, made sure there were no tabs (spaces with the spacebar only), and I redid all the special characters within the description (Kurt once told me that sometimes the wrong format is used, and to use the format within GitHub for those things like commas, hyphens, etc.). Ran it again after all that, and it still pulled the same error. Open to other suggestions. Thank you!

Reply:
Very weird; I copied your file into a new workspace and it was fine with the description.  (There are a bunch of commas missing from band summaries, for example at line 160, but I doubt that would be reflected in a different error check.)